pkg/scaffold/build_dockerfile.go

```diff
@@ -34,9 +34,23 @@ func (s *Dockerfile) GetInput() (input.Input, error) {
 	return s.Input, nil
 }
 
-const dockerfileTmpl = `FROM alpine:3.6
+const dockerfileTmpl = `# Binary builder image
+FROM golang:1.10.3 AS builder
+
+ENV GOPATH /go
+ENV CGO_ENABLED 0
+ENV GOOS linux
+ENV GOARCH amd64
+
+WORKDIR /go/src/{{ .Repo }}
+COPY . /go/src/{{ .Repo }}
+
+RUN go build -o /go/bin/{{ .ProjectName }} {{ .Repo }}/cmd/manager
```

Comment:
Just noticed this, but we need to enable certain environment variables. In the build command, we used `GOOS=linux`, `GOARCH=amd64`, and `CGO_ENABLED=0`. The most important one is `CGO_ENABLED`, because alpine doesn't contain cgo libraries and may not work with the binary produced without that env set.